.github/workflows/ci.yml

```diff
@@ -44,10 +35,29 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python ${{ matrix.python-version }}
+        if: ${{ matrix.os != 'windows-latest' }}
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
 
+      # We use netifaces, which does not ship a 64-bit wheel for the
+      # Python 2.7 + Windows combination, but it ships a 32-bit wheel.
+      # Since MS has removed vcpython27 compiler downloads from their
+      # usual download site, building a netifaces wheel locally is not
+      # an option anymore.  So let us just test with 32-bit Python on
+      # Windows.
+      #
+      # It seems that GitHub Actions' expression syntax is not
```

Comment:
You can do `{{ matrix.os == 'windows-latest' && matrix.python-version == '3.6' }}` I believe.

Reply:
Sadly GitHub Actions skipped right over this:

```
      - name: Set up Python ${{ matrix.python-version }} [Windows x64]
        if: ${{ matrix.os == 'windows-latest' && matrix.python-version == '3.6' }}
        uses: actions/setup-python@v1
        with:
          python-version: ${{ matrix.python-version }}
          architecture: 'x64'
```

... and chose to use the next step instead:

```
      - name: Set up Python ${{ matrix.python-version }} [Windows x86]
        if: ${{ matrix.os == 'windows-latest' && matrix.python-version == '2.7' }}
        uses: actions/setup-python@v1
        with:
          python-version: ${{ matrix.python-version }}
          architecture: 'x86'
```

See https://github.com/tahoe-lafs/tahoe-lafs/pull/1047/checks?check_run_id=2442603808

Reply:
Seems that this works:

```
      - name: Set up Python ${{ matrix.python-version }} [Windows x64]
        if: ${{ ( matrix.os == 'windows-latest' ) && ( matrix.python-version == '3.6' ) }}
        uses: actions/setup-python@v1
        with:
          python-version: ${{ matrix.python-version }}
          architecture: 'x64'

      - name: Set up Python ${{ matrix.python-version }} [Windows x86]
        if: ${{ ( matrix.os == 'windows-latest' ) && ( matrix.python-version == '2.7' ) }}
        uses: actions/setup-python@v1
        with:
          python-version: ${{ matrix.python-version }}
          architecture: 'x86'
```